contrib/PyCuAmpcor/src/SConscript

```diff
@@ -16,6 +17,10 @@ listFiles = ['GDALImage.cu', 'cuArrays.cu', 'cuArraysCopy.cu',
 
 lib = envPyCuAmpcor.SharedLibrary(target = 'PyCuAmpcor', source= listFiles, SHLIBPREFIX='')
 
+# add gdal include path
+gdal_cflags = subprocess.check_output('gdal-config --cflags', shell=True)[:-1].decode('utf-8')
```

Comment:
How does this behave when gdal-config does not exist? I hope it's able to proceed in that case

Reply:
gdal-config is used by scons to check the existence of gdal. So it should be OK. Otherwise, it will be good to define an env['GDAL_PATH'] so that I can pull from.